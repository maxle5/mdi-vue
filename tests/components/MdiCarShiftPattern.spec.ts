
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiCarShiftPattern from "../../src/components/MdiCarShiftPattern.vue";

test("MdiCarShiftPattern snapshot", () => {
  const wrapper = mount(MdiCarShiftPattern, {});
  expect(wrapper.html()).toMatchSnapshot();
});
