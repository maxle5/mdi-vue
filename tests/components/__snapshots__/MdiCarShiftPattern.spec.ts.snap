// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiCarShiftPattern snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M8 5H4V2H8V5M4 22H8V19H4V22M14 2H10V5H14V2M10 22H14V19H10V22M16 2V5H20V2H16M17 11H13V7H11V11H7V7H5V17H7V13H11V17H13V13H19V7H17V11Z" style="fill: currentColor;"></path>
</svg>"
`;
